daemon/inspect_unix.go

```diff
@@ -75,3 +78,25 @@ func addMountPoints(container *Container) []types.MountPoint {
 	}
 	return mountPoints
 }
+
+// populateJSONBase is a platform-specific helper function for inspect that
+// populates the ContainerJSONBase structure
+func populateJSONBase(container *Container, hostConfig *runconfig.HostConfig, containerState *types.ContainerState) *types.ContainerJSONBase {
+
```

Comment:
What this empty lines? :) However I propose to create smaller function like `setLabels`, it'll be more error-prone to changing structs later(and it's changing often).